src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocsummarylocation/InputJavadocSummaryLocationTrailingSpace.java

```diff
@@ -0,0 +1,32 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocsummarylocation;
+
+/**
+ * config:  default
+ * location="SECOND_LINE"
+ */
+public final class InputJavadocSummaryLocationTrailingSpace {
+
+    /**  
+     * ^^ There is a trailing space in the first line. // OK
+     */
+    private int space;
+
+    /**********************
+     * Trailing asterisks in the first line * // OK
+     *
+     **********************/
+    private int asterisks;
+
+    /** ******
+     * There is a space, then leading asterisks in the first line. // OK
+     */
+    private int spaceAsterisk;
+
+    /*** *
```

Comment:
I do not understand difference between `/** **`(OK) and `/*** *` (violation).
Please explain.

Reply:
OK:
```
  v---- Javadoc start
/**  **
     ^- leading asterisks
```
violation:
```
  v---- Javadoc start
/***  *<- any symbols after leading asterisk and spaces
   ^- leading asterisk
```

Each line may start with asterisk(s), followed by spaces. They will be ignored.
For the second case, _after_ the leading asterisk and space, there is one additional symbol.

The Javadoc tool will include this `*` into  the generated documentation.